.gitignore

```diff
@@ -0,0 +1,2 @@
+
+.idea/
```

Comment:
Should we include a wider range of IDE ignores, or should we expect individuals to add their IDE to their global .gitignore?

Reply:
Interesting question, IDE specific gitignore lines would probably best be served in a global .gitignore due to the nature of everybody using different IDE's

Reply:
I'll remove the .gitignore for now and handle it on my local